stream-transformation-test/sample-transformations/src/main/java/uk/gov/sample/event/transformation/SampleArchiveTransformation.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.sample.event.transformation;
+
+import static uk.gov.justice.tools.eventsourcing.transformation.api.TransformAction.ARCHIVE;
+import static uk.gov.justice.tools.eventsourcing.transformation.api.TransformAction.NO_ACTION;
+
+import uk.gov.justice.services.core.enveloper.Enveloper;
+import uk.gov.justice.services.messaging.JsonEnvelope;
+import uk.gov.justice.tools.eventsourcing.transformation.api.EventTransformation;
+import uk.gov.justice.tools.eventsourcing.transformation.api.TransformAction;
+import uk.gov.justice.tools.eventsourcing.transformation.api.annotation.Transformation;
+
+import java.util.stream.Stream;
+
+@Transformation
+public class SampleArchiveTransformation implements EventTransformation {
+
+    private Enveloper enveloper;
+
+    @Override
+    public TransformAction action(final JsonEnvelope event) {
+        if (event.metadata().name().equalsIgnoreCase("sample.archive.events.name")) {
+            return ARCHIVE;
+        }
+        return NO_ACTION;
+    }
+
+
```

Comment:
Doesn't this need to implement apply method?

Or do we now have a default interface method that just returns the existing event